crates/sparrow-instructions/src/evaluators/aggregation/token/collect_struct_token.rs

```diff
@@ -18,7 +29,9 @@ impl StateToken for CollectStructToken {
             let state: CollectStructToken = state;
             self.state = state.state;
         };
-        Ok(())
+
+        // TODO: restore times
+        panic!("time restoration not implemented")
```

Comment:
I'd have to make a new store token and add some logic to the expression executor, so left this out for now.